core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -240,7 +295,7 @@ private void updateStream() throws IOException {
     // Calculate block id.
     long blockId = mStatus.getBlockIds().get(Math.toIntExact(mPosition / mBlockSize));
     // Create stream
-    mBlockInStream = mBlockStore.getInStream(blockId, mOptions);
+    mBlockInStream = mBlockStore.getInStream(blockId, mOptions, lostWorkers);
```

Comment:
if this fails due to excluding all workers, we should indicate that there were available workers, but we hit failures trying to communicate with them.